src/main/java/com/google/jenkins/plugins/computeengine/ComputeEngineCloud.java

```diff
@@ -273,45 +275,14 @@ public void addConfiguration(InstanceConfiguration configuration) {
           break;
         }
 
+        InstanceConfiguration config = configs.get(i % configs.size());
```

Comment:
Shall we add a quick comment here discussing the logic (If one instance configuration doesn't work, it will try the next one in the matched set of configurations)?

Reply:
We can add..

But its not doing anything like that..
It just starts next instances in round robin fashion from available configurations..

So if one of configurations is broken It should eventually start it an another configuration..

Reply:
Ok, can we add a comment like that?